openslides_backend/action/actions/organization/update.py

```diff
@@ -22,65 +23,61 @@ class OrganizationUpdate(
     Action to update a organization.
     """
 
+    group_A_fields = (
+        "name",
+        "description",
+        "legal_notice",
+        "privacy_policy",
+        "login_text",
+        "theme_id",
+        "default_language",
+        "users_email_sender",
+        "users_email_replyto",
+        "users_email_subject",
+        "users_email_body",
+    )
+
+    group_B_fields = (
+        "enable_electronic_voting",
+        "enable_chat",
+        "reset_password_verbose_errors",
+        "limit_of_meetings",
+        "limit_of_users",
+        "url",
+        "saml_enabled",
+        "saml_login_button_text",
+        "saml_attr_mapping",
+    )
+
     model = Organization()
     schema = DefaultSchema(Organization()).get_update_schema(
-        optional_properties=[
-            "name",
-            "description",
-            "legal_notice",
-            "privacy_policy",
-            "login_text",
-            "theme_id",
-            "enable_electronic_voting",
-            "enable_chat",
-            "reset_password_verbose_errors",
-            "limit_of_meetings",
-            "limit_of_users",
-            "url",
-            "users_email_sender",
-            "users_email_replyto",
-            "users_email_subject",
-            "users_email_body",
-            "default_language",
-        ]
+        optional_properties=group_A_fields + group_B_fields,
+        additional_optional_fields={
+            "saml_attr_mapping": {
+                "type": "object",
+                "properties": {
+                    field: {"type": ["string", "null"], "maxLength": 256}
```

Comment:
You could use `optional_str_schema` and insert the `maxLength` into it. I think a `minLength` would be good

Reply:
done